src/events/adapters/importers/fit/importer.fit.ts

```diff
@@ -149,6 +151,23 @@ export class EventImporterFIT {
             activity.intensityZones.push(speedIntensityZones);
           }
 
+          // Add the events
+          fitDataObject.events.filter((activityEvent: FITFileActivityEvent) => {
```

Comment:
For my concern. The purpose of this is to manage Moving/Pause time?

Reply:
As well yes , in case data are written during that time but also to be able to if you want illustrate a pause. 

In some cases especially with HR data you can still have those during a pause.

Reply:
Thanks for the note ;)